.scripts/git-branch.ps1

```diff
@@ -0,0 +1,74 @@
+[CmdletBinding()]
+param
+(
+    [Parameter(Mandatory = $true)]
+    [string]$Name,
+    
+    [Parameter(Mandatory = $true)]
+    [string]$Email
+)
+
+function TagExists($TagName) {
+    
+    git tag | ForEach-Object {
+
+        if ($_ -eq $TagName) {
+            return $true
+        }
+    }
+
+    return $false
+}
+
+$date = Get-Date -UFormat "%Y-%m-%d"
+$branch = "release_$date"
+
+$sources = $env:BUILD_SOURCESDIRECTORY
+
+Write-Output "Set current directory to '$sources'"
+Set-Location $sources
+
+git config user.email $Email
+git config user.name $Name
+
+$message = "Bump package versions for release $date"
+
+Write-Output "Create new branch '$($branch)'"
+
+git checkout -b $branch
+
+$lastCommitId = git rev-list --tags --max-count=1
+$lastTag = git describe $lastCommitId  # By default git describe only shows annotated tags.
+
+if ($LASTEXITCODE -ne 0) {
+
+    git checkout master
+    git branch -d
```

Comment:
>git branch -d [](start = 4, length = 13)

IIRC git branch takes a branch name as a parameter. if it can function with the current branch, then you're deleting master. in both cases i think this is dangerous. you probably want to delete the branch first (by specifying $branch), THEN checkout master